traffic_ops/traffic_ops_golang/crconfig/servers.go

```diff
@@ -190,10 +191,12 @@ select s.host_name, ds.xml_id
 from deliveryservice_server as dss
 inner join server as s on dss.server = s.id
 inner join deliveryservice as ds on ds.id = dss.deliveryservice
+inner join type as dt on dt.id = ds.type
 inner join profile as p on p.id = s.profile
 inner join status as st ON st.id = s.status
 where ds.cdn_id = (select id from cdn where name = $1)
-and ds.active = true
+and ds.active = true` +
+		fmt.Sprintf(" and dt.name != '%s' ", tc.DSTypeAnyMap) + `
```

Comment:
Nitpick: `+ string(tc.DSTypeAnyMap) +` is both easier to read and faster

Reply:
Agreed. I think I did it this way because it's how the other ANY_MAP query fix was done too.